tests/lib/rules/no-fallthrough.js

```diff
@@ -310,6 +439,83 @@ ruleTester.run("no-fallthrough", rule, {
                     column: 2
                 }
             ]
+        },
+        {
+            code: `
+switch (foo) {
+    case 0:
+        a();
+        break;
+    /* falls through */
+    case 1:
+        b();
+}`,
+            options: [{ reportUnusedFallthroughComment: true }],
+            errors: [
+                {
+                    line: 6,
+                    messageId: "unusedFallthroughComment"
+                }
+            ]
+        },
+        {
+            code: `
+switch (foo) {
+    default:
+        a();
+        break;
+    /* falls through */
+    case 1:
+        b();
+}`,
+            options: [{ reportUnusedFallthroughComment: true }],
+            errors: [
+                {
+                    line: 6,
+                    messageId: "unusedFallthroughComment"
+                }
+            ]
+        },
+        {
+            code: `
+switch(foo){
+    case 1:
+        doSomething();
+        break;
+    // falls through
+    case 2: doSomething();
+}`,
+            options: [{ reportUnusedFallthroughComment: true }],
+            errors: [
+                {
+                    line: 6,
+                    messageId: "unusedFallthroughComment"
+                }
+            ]
+        }, {
```

Comment:
Can you also add a test where a fallthrough comment in a block is reported, for example:

```js
        {
            code: `
switch (foo) {
    case 0: {
        a();
        break;
        // falls through
    }
    case 1:
        b();
}`,
            options: [{ reportUnusedFallthroughComment: true }],
            errors: [
                {
                    line: 6,
                    messageId: "unusedFallthroughComment"
                }
            ]
        },
```

Reply:
makes sense! added